src/test/java/com/google/sps/integration/ElectionInfoIT.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.integration;
```

Comment:
Adding a skeleton file with no tests (yet) is fine, but it'd be nice to have put this in its own PR. Prefer small PRs because they're easier to review. 

See:
https://drive.google.com/file/d/1CNUwfkvB00ZPeAb7aVssF9mIYINzumeZ/view?usp=sharing

Reply:
Good to know, thanks for linking the doc :)